tests/python/pants_test/releases/test_reversion.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import glob
+import os
+import shutil
+
+import requests
+
+from pants.util.contextutil import temporary_dir
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class ReversionTest(PantsRunIntegrationTest):
+
+  def test_run(self):
+    with temporary_dir() as dest_dir:
+      # Download an input whl.
+      # TODO: Not happy about downloading things. Attempted to:
+      #  ./pants setup-py bdist_wheel -w $dest_dir
```

Comment:
Will fix before landing, but this was more like:
```
./pants setup-py --run="bdist_wheel" examples/src/thrift/org/pantsbuild/example/distance:distance-python
```
...which, when executed directly, results in a whl at:
```
dist/pantsbuild.pants.distance-thrift-python-0.0.1/dist/pantsbuild.pants.distance_thrift_python-0.0.1-py2-none-any.whl
```

When executed in the context of a test, the command completes successfully with the same stdout, but with no whl created.